lib/bmp280/sensor/bmp180_sensor.ex

```diff
@@ -0,0 +1,47 @@
+defmodule BMP280.BMP180Sensor do
+  @moduledoc false
+
+  alias BMP280.{BMP180Calibration, BMP180Comm, Calc, Measurement}
+
+  @behaviour BMP280.Sensor
+
+  @impl true
+  def init(%{sensor_type: :bmp180, transport: transport} = state) do
+    with {:ok, calibration_binary} <- BMP180Comm.read_calibration(transport),
+         calibration <- BMP180Calibration.from_binary(calibration_binary),
+         do: %{state | calibration: calibration}
+  end
+
+  @impl true
+  def read(%{transport: transport} = state) do
+    BMP180Comm.set_temperature_reading(transport)
```

Comment:
```suggestion
    :ok = BMP180Comm.set_temperature_reading(transport)
```

so we can raise on failure